gwswlib/scripts.py

```diff
@@ -17,11 +19,13 @@ def get_parser():
         default=False,
         help="Verbose output",
     )
-    # add arguments here
-    # parser.add_argument(
-    #     'path',
-    #     metavar='FILE',
-    # )
+    parser.add_argument(
+        "--hydx",
+        default=None,
+        metavar="hydxpath",
```

Comment:
In het Engels zijn het twee woorden, dus (ook i/d rest v/d code) graag `hydx_path`?